imgui-core/src/main/kotlin/imgui/demo/showExampleApp/Log.kt

```diff
@@ -128,16 +129,14 @@ object Log {
                         textEx(line)
                 }
             else {
-                textEx(buf.toString())
-                //TODO: Fix and remove above line
-                /*val clipper = ListClipper()
-                clipper.begin(lineOffsets.size)
+                val clipper = ListClipper(lineOffsets.size)
                 while(clipper.step()) {
-                    for(line_no in clipper.display.start until clipper.display.endInclusive - 1) {
-                        textUnformatted(buf.subSequence(lineOffsets[line_no], if(line_no + 1 < lineOffsets.size) lineOffsets[line_no + 1] - 1 else buf.length).toString())
+                    for(line_no in clipper.display.first until clipper.display.last) {
```

Comment:
We can directly `for(line_no in clipper.display)`

Reply:
Nice catch.